rpm_spec/subpackages/manageiq-core

```diff
@@ -31,6 +31,9 @@ Requires: shadow-utils
 /usr/bin/id manageiq > /dev/null 2>&1 || /usr/sbin/useradd --system --create-home manageiq
 # create a manageiq home directory if it doesn't exist
 mkdir -p /home/manageiq && chown manageiq:manageiq /home/manageiq
+# Add sub uids/gids for running non-privileged containers
+grep manageiq /etc/subuid >/dev/null 2>&1 || /usr/sbin/usermod --add-subuids 100000-165535 manageiq
+grep manageiq /etc/subgid >/dev/null 2>&1 || /usr/sbin/usermod --add-subgids 100000-165535 manageiq
```

Comment:
NOTE this should probably live in manageiq-appliance since this is only needed for podman but I wanted to make sure `-core` was going to run before `-appliance` in case the manageiq user wasn't created yet.

Reply:
good job with the usermod instead of relying upon useradd.

Reply:
Okay for testing the move to manageiq-appliance I tried deleting my manageiq user, checked the subuid/subgid files and they were cleared automatically (nice job userdel), and reinstalled the rpm packages.  The manageiq user creation was in fact run first and the subuid/subgid files were created properly